src/xsar/sentinel1_meta.py

```diff
@@ -831,6 +831,14 @@ def coords2heading(self, atracks, xtracks, to_grid=False, approx=True):
         _, heading = haversine(lon1, lat1, lon2, lat2)
         return heading
 
+    @property
+    def bursts(self):
+        if self.xml_parser.get_var(self.files['annotation'].iloc[0], 'annotation.number_of_bursts') > 0:
+            return self.xml_parser.get_compound_var(self.files['annotation'].iloc[0], 'bursts')
+        else:
+            # no burst, return empty dataframe with 'burst' column
+            return pd.DataFrame({'burst': []})
```

Comment:
if `bursts` stucture is a xarray dataset, no bursts should also be a empty xarray

Reply:
DataFrame replaced by xarray.Dataset.
![image](https://user-images.githubusercontent.com/14925067/155685903-2e3a1538-9906-47bf-a5e0-314bbbee7127.png)
![image](https://user-images.githubusercontent.com/14925067/155685953-7ef02409-733d-4f81-ad06-8d83bfbce1a7.png)